### DISCUSSION TITLE
How to fix this line?

### DISCUSSION THREAD

**Discussion by Ser4ph4** [Godiesc/firefox-gx#223] [General]
Hello, how do I fix this? This little flaw in the line that doesn't touch the edge.
![noname](https://github.com/user-attachments/assets/aaf25781-0cfe-4e02-a71f-bbbf5ea437f0)

**Comment by Godiesc** [comment]
hi,
that line was designed in that way. You can change that margin adding the next code in `ogx_tricks.css`:

```
menuseparator::before {
    margin-inline: -12px !important;
}
```

**Reply by Ser4ph4** [reply]
Perfect, it worked :)
Thanks friend

**Comment by Ser4ph4** [comment]
I identified one more point, in downloads
![noname](https://github.com/user-attachments/assets/442cda1d-2af9-495e-9baf-a842926dc141)

**Reply by Godiesc** [reply]
contextual menus are diff of arrowpanels, this code is for that elements, you can play with the size:

```
/* Gradient line - separators */

toolbarseparator {
    margin-inline: 6px !important;
}

#PlacesToolbar menupopup[placespopup] menuseparator {
    margin-inline: 4px !important;
}
```

**Reply by Ser4ph4** [reply]
> contextual menus are diff of arrowpanels, this code is for that elements, you can play with the size:
> 
> ```
> /* Gradient line - separators */
> 
> toolbarseparator {
>     margin-inline: 6px !important;
> }
> 
> #PlacesToolbar menupopup[placespopup] menuseparator {
>     margin-inline: 4px !important;
> }
> ```

Perfect, I got it here
Thanks
